### PR TITLE
Support messages for video calls and key change

### PR DESCRIPTION
**Pull request by ericthegrey**
This PR adds additional message types:
- Video calls (incoming, outgoing, missed)
- Key changes (i.e. "the security number of a participant changed"

### CODE REVIEW

File: signal2html/html.py [hunk 1]
```diff
@@ -26,6 +26,8 @@
     is_missed_call,
     is_outgoing_call,
     is_group_call,
+    is_video_call,
+    is_key_update,
     is_group_ctrl,
     is_group_v2_data,
```
Comment (GjjvdBurg) [inline signal2html/html.py:32]: ``is_video_call`` and ``is_group_v2_data`` seem to be unused

**Review by GjjvdBurg — APPROVED** [review]
Looks good to me! I had one minor comment regarding unused imports, otherwise it's great!

Note that I did test this on my v23 database and this didn't have a video call message type, so this is probably a later addition. Doesn't affect anything in this PR, but may be good background info.

File: signal2html/html.py (hunk 1 above)
Comment (ericthegrey) [reply signal2html/html.py]: Indeed, removed

**Comment by ericthegrey** [comment]
Hi @GjjvdBurg , thanks for reviewing!

**Comment by GjjvdBurg** [comment]
Thanks for merging! :+1: I think I'll prepare a release once we have #30 sorted out as well

**Comment by ericthegrey** [comment]
Sounds good :)

**Pull request merged**